### PR TITLE
Edit Parsers, AddCommand, EditCommand and refactor tests

### PR DESCRIPTION
**Pull request by mannggoo**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -1,60 +1,54 @@
...
+            + "description "
+            + "<Optional> " + PREFIX_DATETIME + "Deadline "
+            + "<Optional> " + PREFIX_REMINDER + "Reminder "
+            + "<Optional> " + PREFIX_PRIORITY + "Priority "
+            + "<Optional> " + PREFIX_TAG + "Tag ";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New Item added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This item already exists in the List";
```
Comment (sianghwee) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: Hi, perhaps change this to MESSAGE_DUPLICATE_ITEM?

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -26,11 +23,15 @@ public FindCommand(NameContainsKeywordsPredicate predicate) {
...
-    public CommandResult execute(Model model) {
+    public CommandResult execute(ItemModel model) {
+        /* Original Code
         requireNonNull(model);
         model.updateFilteredPersonList(predicate);
         return new CommandResult(
                 String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+
+         */
+        return new CommandResult("dummy");
```
Comment (sianghwee) [inline src/main/java/seedu/address/logic/commands/FindCommand.java]: The find logic is actually already implemented in item list, but I can implement the find command as well :)

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (mannggoo) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Oh nice catch, thanks!

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 2 above)
Comment (mannggoo) [reply src/main/java/seedu/address/logic/commands/FindCommand.java]: Sure, that would be great :)

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -200,27 +240,5 @@ public void setTags(Set<Tag> tags) {
         public Optional<Set<Tag>> getTags() {
             return (tags != null) ? Optional.of(Collections.unmodifiableSet(tags)) : Optional.empty();
         }
-
-        @Override
-        public boolean equals(Object other) {
```
Comment (blimyj) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: Do you want to override the equals method?

**Pull request merged**